### PR TITLE
Fix deprecation in tree builder for Symfony 4.2+

### PR DESCRIPTION
**Pull request by alcaeus**
See https://github.com/doctrine/DoctrineBundle/pull/853 - I forgot this in the previous 1.3 release but it should indeed be fixed in 1.3 before we drop support for old stuff for 1.4.

### CODE REVIEW

**Comment by alcaeus** [comment]
Disabled `phpcs` for this old version:
> Do not use the else or elseif keywords

We'll be introducing Doctrine CS in master soon, so no need for a weird coding standard here.

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -87,7 +87,14 @@ public function getConfigTreeBuilder()
     {
         $self            = $this;
         $builder         = new TreeBuilder();
```
Comment (xabbuh) [inline DependencyInjection/Configuration.php]: here, `doctrine_cache` needs to be passed as the first argument

**Comment by alcaeus** [comment]
I noticed that @javiereguiluz created #143 a while back, but builds there fail. Seems like I forgot a couple of instances as well, so I'll clean this up later today.

**Comment by alcaeus** [comment]
Build failures on PHP 5.3 and 5.5 are unrelated to this change, will investigate them separately.

**Pull request merged**